src/lambda-manager/README.md

```diff
@@ -1,22 +1,87 @@
+This Lambda Function was created pick up newly created and existing log groups and attach them to Firehose or Lambda integration
 
 Environment variables:
 
 | Parameter | Description | Default Value | Required |
 |---|---|---|---|
-| CoralogixRegion | The Coralogix location region, possible options are [Europe, Europe2, India, Singapore, US, US2] | Europe | :heavy_check_mark: |
-| ApiKey | The Coralogix private key which is used to validate your authenticity | | :heavy_check_mark: |
-| RegexPattern | Set up this regex to match the Log Groups names that you want to automatically subscribe to Coralogix Firehose.| | :heavy_check_mark: |
-| LogsFilter | Subscription filter to select which logs needs to be sent to Coralogix. Default is for Lambda Errors that are not sendable by Coralogix Lambda Layer. | | :heavy_check_mark: |
-| FIREHOSE_ARN | Arn for the firehose to subscribe the log groups (By default is the firehose created by Serverless Template) | | :heavy_check_mark: |
-| FIREHOSE_ROLE | Arn for the role to allow firehose subscription (By default is the role created by Serverless template) | | :heavy_check_mark: |
+| RegexPattern | Set up this regex to match the Log Groups names that you want to automatically subscribe to the destination| | :heavy_check_mark: |
+| LogsFilter | Subscription filter to select which logs needs to be sent to Coralogix. For Example for Lambda Errors that are not sendable by Coralogix Lambda Layer '?REPORT ?"Task timed out" ?"Process exited before completing" ?errorMessage ?"module initialization error:" ?"Unable to import module" ?"ERROR Invoke Error" ?"EPSAGON_TRACE:"'. | | :heavy_check_mark: |
+| DESTINATION_ARN | Arn for the firehose to subscribe the log groups (By default is the firehose created by Serverless Template) | | :heavy_check_mark: |
+| DESTINATION_ROLE | Arn for the role to allow destination subscription to be pushed (Lambda or Firehose) | | :heavy_check_mark: |
+| DESTINATION_TYPE | Type of destination (Lambda or Firehose) | | :heavy_check_mark: |
+| SCAN_OLD_LOGGROUPS | This will scan all LogGroups in the account and apply the subscription configured, will only run Once and set to false. Default is false | false | :heavy_check_mark: |
 | FunctionArchitecture | Lambda function architecture, possible options are [x86_64, arm64] | x86_64 | |
 | FunctionMemorySize | The maximum allocated memory this lambda may consume. Default value is the minimum recommended setting please consult coralogix support before changing. | 1024 |  |
 | FunctionTimeout | The maximum time in seconds the function may be allowed to run. Default value is the minimum recommended setting please consult coralogix support before changing. | 300 |  |
 | NotificationEmail | Failure notification email address | | |
 
+## Requirements
+
+### Firehose
+
+We are asuming you deployed ouf Firehose integration per our integration https://coralogix.com/docs/aws-firehose/
```

Comment:
Spalling
We are **_assuming_** you deployed our Firehose integration per **_our_** integration https://coralogix.com/docs/aws-firehose/